src/html.sortable.js

```diff
@@ -682,7 +692,10 @@ var sortable = function (sortableElements, options) {
 
     // Handle dragover and dragenter events on draggable items
     var onDragOverEnter = function (e) {
-      if (!dragging || !_listsConnected(sortableElement, dragging.parentElement)) {
+      if (!dragging || !_listsConnected(sortableElement, dragging.parentElement) || _data(sortableElement, '_disabled') === 'true') {
+        return
+      }
+      if (maxItems && _filter(_getChildren(sortableElement), _data(sortableElement, 'items')).length >= maxItems) {
```

Comment:
This is because of the placeholder class getting added. If you pass an "items" option that doesn't match the placeholder, it works correctly this way. I think that's better than having it always off-by-one which seems confusing.